salt/runners/cache.py

```diff
@@ -310,20 +310,28 @@ def clear_git_lock(role, remote=None, **kwargs):
         have their lock cleared. For example, a ``remote`` value of **github**
         will remove the lock from all github.com remotes.
 
-    type : update,checkout
-        The types of lock to clear. Can be ``update``, ``checkout``, or both of
-    et (either comma-separated or as a Python list).
+    type : update,checkout,mountpoint
+        The types of lock to clear. Can be one or more of ``update``,
+        ``checkout``, and ``mountpoint``, and can be passed either as a
+        comma-separated or Python list.
 
         .. versionadded:: 2015.8.8
+        .. versionchanged:: 2017.7.3
```

Comment:
This should be changed to `Oxygen` since this PR was moved to `develop`.

Reply:
Fixed.